kustomize/base/shard/resources/bridgeingresses.com.redhat.service.bridge-v1.yml

```diff
@@ -0,0 +1,51 @@
+# Generated by Fabric8 CRDGenerator, manual edits might get overwritten!
```

Comment:
What is the lifecycle of this file? Is it generated and then manually copied here? How/when is it updated?

Reply:
Yes: it is generated and it was manually copied here. Some weeks ago I created a ticket to add a job in the CI so to check that those files are in sync https://issues.redhat.com/browse/MGDOBR-97